Source/WebCore/platform/graphics/ContentsFormat.cpp

```diff
@@ -45,7 +42,11 @@ std::optional<DestinationColorSpace> contentsFormatExtendedColorSpace(ContentsFo
 #endif
 #if ENABLE(PIXEL_FORMAT_RGBA16F) && ENABLE(DESTINATION_COLOR_SPACE_EXTENDED_REC_2020)
```

Comment:
If you stop using Rec2020 below it's' weird to still have this `#if ...DESTINATION_COLOR_SPACE_EXTENDED_REC_2020`

Reply:
Fixed.